vllm/v1/attention/backends/flex_attention.py

```diff
@@ -465,8 +508,8 @@ def forward(
 
         out = flex_attention_compiled(
             query,
-            key_cache,
-            value_cache,
+            key_tensor,
+            value_tensor,
```

Comment:
I've renamed the variables here because in one case they are key and value and in the other case key_cache and key_value.